public/styles.css

```diff
@@ -0,0 +1,36 @@
+body{
+height:100vh;
+}
+.font-bold {
+font-weight: bold;
+}
+ .brush{
+content: "";
+background-image: url(images/brush-stroke.svg);
+background-repeat: no-repeat;
+background-position: -300px 160%;
```

Comment:
🗣️ This -300px is interesting, what's going on there out of curiosity?

Reply:
positioning the brush stroke in the right place